scipy/signal/tests/test_signaltools.py

```diff
@@ -866,9 +866,9 @@ def test_basic(self):
         assert_array_almost_equal(signal.wiener(g), h, decimal=6)
         assert_array_almost_equal(signal.wiener(g, mysize=3), h, decimal=6)
 
+padtype_options = ["constant", "mean", "median", "minimum", "maximum", "line"]
```

Comment:
```suggestion
padtype_options = ["constant", "mean", "median", "minimum", "maximum", "line"]

```
PEP8 is causing the Travis failure

https://travis-ci.org/scipy/scipy/jobs/568088499

I assume the line number is off because Travis builds a merged version of the PR
```
scipy/signal/tests/test_signaltools.py:878:1: E305 expected 2 blank lines after class or function definition, found 1
```

Reply:
This is addressed now. Travis still fails, but it seems it its a network connection problem now